pages/rbi/usecases.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Range Based Incentive uses cases
+description: Range Based Incentive usecases.
+---
+
+### Stablecoin Pools
```

Comment:
Same here and in general, I'd avoid unnecessary capital letters if it's not an acronym.